### PR TITLE
S04 level rome

### PR DESCRIPTION
**Pull request by Romlion**

### CODE REVIEW

File: frontend/src/components/LevelPage/RomePage/RomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          imgPopUpClass={cicero.imgPopUpClass}
+        />
+        <TargetButtonRome
+          data={auguste.data}
+          img={Auguste}
+          imgClasse={auguste.imgClasse}
+          item={auguste.item}
+          imgPopUpClass={auguste.imgPopUpClass}
+        />
+        <TargetButtonRome
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/RomePage/RomePage.jsx]: No item here ?

File: frontend/src/components/LevelPage/RomePage/RomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          data={spartacus.data}
+          img={Spartacus}
+          imgClasse={spartacus.imgClasse}
+          imgPopUpClass={spartacus.imgPopUpClass}
+        />
+        <TargetButtonRome
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/RomePage/RomePage.jsx]: No Item here ?

File: frontend/src/components/LevelPage/RomePage/RomePage.jsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          data={pompey.data}
+          img={Pompey}
+          imgClasse={pompey.imgClasse}
+          imgPopUpClass={pompey.imgPopUpClass}
+        />
+        <TargetButtonRome
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/RomePage/RomePage.jsx]: No item here ?

**Review by Cyrildab — APPROVED** [review]
approved !

File: frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  overflow: hidden;
+  &:hover {
+    filter: drop-shadow(3px 5px 2px rgb(0 0 0 / 0.2));
+    cursor: pointer;
+  }
+}
+
+.juliusCaesarPopUp {
+  width: 10vw;
+  margin: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss]: No vw in height

File: frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    filter: drop-shadow(3px 5px 2px rgb(0 0 0 / 0.3));
+    cursor: pointer;
+  }
+}
+
+.pompey {
+  width: 4vw;
+  position: absolute;
+  top: 6%;
+  left: 77%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss]: Right is better

File: frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+.pompeyPopUp {
+  width: 7vw;
+  margin: 1rem;
+}
+
+.vercingetorix {
+  width: 12vw;
+  position: absolute;
+  bottom: 5%;
+  right: 55%;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss]: Left is better here

File: frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss (hunk 4 above)
Comment (Romlion) [reply frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss]: This probleme is fix

File: frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss (hunk 5 above)
Comment (Romlion) [reply frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss]: This problem to

File: frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss (hunk 6 above)
Comment (Romlion) [reply frontend/src/components/LevelPage/TargetButton/TargetButtonRome.scss]: and this to

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**